backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/steps/FilterDuplicates.kt

```diff
@@ -23,6 +23,8 @@ class FilterDuplicates(private val logger: Logger) : PipelineStep<List<Accepting
         val store = last()
 
         logger.logRemoveDuplicates(store, size - 1)
+        val issuers = map { "'${it.districtName ?: it.freinetId.toString()}'" }.toSet().joinToString()
```

Comment:
Is this the same as 
```kotlin
        val issuers = it.districtName ?: it.freinetId.toString()
```
?

Reply:
No, this lists the freinetIds/districtNames of all duplicates. So it is something like `'508', '963'`, `'Landkreis Fürth', '766'` or `'Landkreis Bad Tölz-Wolfratshausen'`, i.e. multiple freinetIds and districtNames for logging purposes.

Reply:
Ah, `map` calls "this.map"... I was confused because of the syntax :) Thanks for the clarification.